plenum/server/primary_elector.py

```diff
@@ -357,7 +357,7 @@ def processPrimary(self, prim: Primary, sender: str) -> None:
                     logger.debug("{} selected primary on the basis of {}".
```

Comment:
basically I want the fixture poolNodesStarted to only finish when a primary has been elected.